Unix/share/omischema/CIM-2.32.0/Device/CIM_NetworkAdapter.mof

```diff
@@ -12,7 +12,7 @@
     Abstract, Version ( "2.10.0" ), 
     UMLPackagePath ( "CIM::Device::NetworkAdapter" ), 
     Description ( 
-       "Note: The use of the CIM_NetworkAdpater class has been "
+       "Note: The use of the CIM_NetworkAdapter class has been "

```

Comment:
oh, if this entire directory tree is https://www.dmtf.org/standards/cim/cim_schema_v2320 and there's a proper way to ask them to fix stuff, i'm happy to send things to them. if the only way to do so is as a member, then, if MS is a member and could do it, that'd be great.

Obviously, if there's an upstream source for a thing, it's probably best to drop the changes from this pr, just let me know. and in general, i'm happy to send changes to an upstream.

*LLVM has been on my todo list for a long time, it's still there, but it's a big thing.